src/Redux/reducers/ErrorReducer.js

```diff
@@ -34,6 +36,18 @@ const errorReducer = (state = InitialErrorState, action) => {
     case errorActions.RESET_REQUEST_DETAILS_ERROR: {
       return { ...state, ...action.payload };
     }
+    case errorActions.UPDATE_REQUEST_SOLUTIONS_ERROR: {
+      return { ...state, ...action.payload };
+    }
+    case errorActions.RESET_REQUEST_SOLUTIONS_ERROR: {
+      return { ...state, ...action.payload };
+    }
+    case errorActions.UPDATE_REQUEST_STAKES_ERROR: {
+      return { ...state, ...action.payload };
+    }
+    case errorActions.RESET_REQUEST_STAKES_ERROR: {
+      return { ...state, ...action.payload };
+    }
     default: {
```

Comment:
why to update all the errors for every action. It could be more specific

Reply:
Will be part of the next enhancements.